lte/gateway/c/core/oai/lib/hashtable/hashtable.h

```diff
@@ -235,6 +235,8 @@ hashtable_rc_t hashtable_uint64_ts_dump_content(
 hashtable_rc_t hashtable_uint64_ts_insert(
     hash_table_uint64_ts_t* const hashtbl, const hash_key_t key,
     const uint64_t dataP);
+hashtable_rc_t hashtable_uint64_ts_free(
+    hash_table_uint64_ts_t* const hashtbl, const hash_key_t key);
```

Comment:
check out `_remove` functions. you do not need to add new functions to the hashtable library.

Reply:
removed hashtable_uint64_ts_free and replaced caller APIs with hashtable_uint64_ts_remove